spec/lib/puppet_spec/matchers.rb

```diff
@@ -1,18 +1,5 @@
 require 'stringio'
 
-########################################################################
-# Backward compatibility for Jenkins outdated environment.
-module RSpec
-  module Matchers
-    module BlockAliases
-      alias_method :to,     :should      unless method_defined? :to
-      alias_method :to_not, :should_not  unless method_defined? :to_not
-      alias_method :not_to, :should_not  unless method_defined? :not_to
```

Comment:
This change seems unrelated to dropping rspec-collections?

Reply:
I don't care so much, just glad to not be monkey patching rspec

Reply:
When I removed the collections, this flagged up as error and the tests are passing without so :arrow_right: :wastebasket: